### DISCUSSION TITLE
A little help flashing sponsor version

### DISCUSSION THREAD

**Discussion by Celticpure** [alf45tar/PedalinoMini#405] [Q&A]
I am having trouble flashing the sponsor version with Visual Studio Code.

 Even when I set the port no. and flash it, it will not then load the system image. At this point VSC says it cannot access the port. I have tried it many times and it doesn't work. Version 2.5.2 flashes no problem. 

Is there something I am missing?

**Comment by alf45tar** [comment]
Check it the serial port is not used by other application or VSC monitor itself.

**Comment by Celticpure** [comment]
Ok got it sorted. I had to erase flash in VSC first then upload and upload system image then. all working now.
